components/gantry/singleaxis/singleaxis.go

```diff
@@ -456,19 +469,34 @@ func (g *singleAxis) Lengths(ctx context.Context, extra map[string]interface{})
 }
 
 // MoveToPosition moves along an axis using inputs in millimeters.
-func (g *singleAxis) MoveToPosition(ctx context.Context, positions []float64, extra map[string]interface{}) error {
+func (g *singleAxis) MoveToPosition(ctx context.Context, positions, speeds []float64, extra map[string]interface{}) error {
 	ctx, done := g.opMgr.New(ctx)
 	defer done()
 
 	if len(positions) != 1 {
 		return fmt.Errorf("MoveToPosition needs 1 position to move, got: %v", len(positions))
 	}
 
+	if len(speeds) != 1 {
+		return fmt.Errorf("MoveToPosition needs 1 speed to move, got: %v", len(speeds))
```

Comment:
```suggestion
		return fmt.Errorf("singleAxis MoveToPosition needs 1 speed to move, got: %v", len(speeds))
```

Reply:
should I change the other error for position lengths as well?